From b1d46cfa168d2472f6006f87c3b52f8ba396e224 Mon Sep 17 00:00:00 2001
From: Dennis Cagle <dcagle@codeaurora.org>
Date: Fri, 2 Nov 2018 17:03:28 -0700
Subject: [PATCH] Move root user to unconfined context

Start the root user in unconfined context instead of sysadm

Change-Id: I27216a10609a02fc11f4e437fefd66a3ea774298
---
 policy/users | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/policy/users b/policy/users
index ca20375..a473dfa 100644
--- a/policy/users
+++ b/policy/users
@@ -43,7 +43,7 @@ ifdef(`direct_sysadm_daemon',`
 # not in the sysadm_r.
 #
 ifdef(`direct_sysadm_daemon',`
-	gen_user(root, sysadm, sysadm_r staff_r ifdef(`enable_mls',`secadm_r auditadm_r') system_r, s0, s0 - mls_systemhigh, mcs_allcats)
+	gen_user(root, unconfined, unconfined_r sysadm_r staff_r ifdef(`enable_mls',`secadm_r auditadm_r') system_r, s0, s0 - mls_systemhigh, mcs_allcats)
 ',`
-	gen_user(root, sysadm, sysadm_r staff_r ifdef(`enable_mls',`secadm_r auditadm_r'), s0, s0 - mls_systemhigh, mcs_allcats)
+	gen_user(root, unconfined, unconfined_r sysadm_r staff_r ifdef(`enable_mls',`secadm_r auditadm_r'), s0, s0 - mls_systemhigh, mcs_allcats)
 ')
-- 
1.9.1

